system/containers.toit

```diff
@@ -176,10 +189,14 @@ class ContainerImageFlash extends ContainerImage:
       manager.image-registry.free allocation
 
 abstract class ContainerServiceProvider extends ServiceProvider
-    implements ContainerService ServiceHandler:
+    implements ContainerService ContainerEventService ServiceHandler:
+  // These event constants should be kept in sync with the ones in lib/system/containers.toit.
+  static EVENT-BACKGROUND-STATE-CHANGE ::= 0
```

Comment:
Remind me why you can't use that constant instead?

Reply:
done.